### PR TITLE
Branch 2019 r2 roles

### PR DESCRIPTION
**Pull request by YuriyZaletskyy**
Hi,
I've created code, that allows to add users based on their roles. Will be thankful for review.

### CODE REVIEW

File: PX.Survey.Ext/SurveyMaint.cs [hunk 1]
```diff
@@ -1,37 +1,48 @@
...
+using PX.Data.Access.ActiveDirectory;
 using PX.Data.BQL;
 using PX.Data.BQL.Fluent;
 using PX.Objects.CR;
 using PX.Objects.CS;
+using PX.SM;
+using PX.Survey.Ext;
 
-namespace PX.Survey.Ext
+namespace AcumaticaSurveysLibr
```
Comment (markoan) [inline PX.Survey.Ext/SurveyMaint.cs:16]: Is this the new official namespace?

File: PX.Survey.Ext/SurveyMaint.cs [hunk 2]
```diff
@@ -1,37 +1,48 @@
...
+
         [PXViewName(PX.Objects.CR.Messages.Attributes)]
         public CSAttributeGroupList<Survey.surveyID, SurveyCollector> Mapping;
 
         public SelectFrom<SurveyUser>.Where<SurveyUser.surveyID.IsEqual<Survey.surveyID.FromCurrent>>.View SurveyUsers;
 
-        [PXHidden]
-        [PXCopyPasteHiddenView]
-        public PXSetup<SurveySetup> SurveySetup;
+        [PXHidden] [PXCopyPasteHiddenView] public PXSetup<SurveySetup> SurveySetup;
```
Comment (markoan) [inline PX.Survey.Ext/SurveyMaint.cs:34]: Is this formating in a single line by mistake?

**Review by markoan — COMMENTED** [review]
Have some questions about the namespace change and some formating.

File: PX.Survey.Ext/SurveyMaint.cs (hunk 2 above)
Comment (YuriyZaletskyy) [reply PX.Survey.Ext/SurveyMaint.cs:34]: Kind of strange, as in Visual Studio I see multiple lines.

File: PX.Survey.Ext/SurveyMaint.cs (hunk 1 above)
Comment (YuriyZaletskyy) [reply PX.Survey.Ext/SurveyMaint.cs:16]: Good question. As far as I see in history of commits, I wasn't initiator of renaming change. For example commit from @DhirenChhapgar  also had namespace PX.Survey.Ext, also I wonder why only my commit rendered as the one changing namespace

File: PX.Survey.Ext/SurveyMaint.cs (hunk 2 above)
Comment (YuriyZaletskyy) [reply PX.Survey.Ext/SurveyMaint.cs:34]: image

File: PX.Survey.Ext/SurveyMaint.cs (hunk 1 above)
Comment (DhirenChhapgar) [reply PX.Survey.Ext/SurveyMaint.cs:16]: @YuriyZaletskyy , is it rendering wring here and is correct in your source? We need this fix

**Pull request merged**